src/05_lists.py

```diff
@@ -1,29 +1,35 @@
-# For the exercise, look up the methods and functions that are available for use
-# with Python lists.
+# For the exercise, look up the methods and functions
+# that are available for use with Python lists.
 
 x = [1, 2, 3]
 y = [8, 9, 10]
 
 # For the following, DO NOT USE AN ASSIGNMENT (=).
 
 # Change x so that it is [1, 2, 3, 4]
-# YOUR CODE HERE 
+# YOUR CODE HERE
+x.append(4)
 print(x)
 
 # Using y, change x so that it is [1, 2, 3, 4, 8, 9, 10]
-# YOUR CODE HERE 
+# YOUR CODE HERE
+x.extend(y)
 print(x)
 
 # Change x so that it is [1, 2, 3, 4, 9, 10]
-# YOUR CODE HERE 
+# YOUR CODE HERE
+del x[4]
```

Comment:
.remove() is an option here as well (but del does have better performance if you know the index ahead of time)

Reply:
Thanks! That's handy, will keep that in mind. :)